src/frontend/src/components/TaskList.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Box, Tab, Tabs, Typography } from '@mui/material';
+import { useState } from 'react';
+import PageBlock from '../layouts/PageBlock';
+import { NERButton } from './NERButton';
+import { useAuth } from '../hooks/auth.hooks';
+import { Add } from '@mui/icons-material';
+import { Auth } from '../utils/types';
+
+interface TaskListProps {
+  defaultClosed?: boolean;
+}
+
+// Page block containing task list view
+const TaskList = ({ defaultClosed }: TaskListProps) => {
+  const auth: Auth = useAuth();
+  const taskListTitle: string = 'Task List';
+  const addTaskButton: React.ReactNode = (
```

Comment:
can u make this button 32 px tall, so it's the same size as the text on the other end of the pageblock, and doesn't make the pageblock thicker when collapsed, might need to play with the width as well to get it to look nice; also imo move the addTaskButton code below handleTabChange, just so all the ts logic is at the top and all the react components are near the bottom all grouped up